profile/README.md

```diff
@@ -4,3 +4,57 @@ Code.org is an education innovation nonprofit dedicated to the vision that every
+
+### 🏗️ Infrastructure Team
+
+Our Infrastructure Team provides the backbone for the platform that supports our students and teachers. This includes our hosting infrastructure and related services (we use AWS), data management, security programs, developer and deployment services and much more. This team also owns our scaling and provisioning systems, which are critical during Hour of Code, where we invite every student in the world to complete an hour of coding with us.
+
+### Frequently Asked Questions
+
+#### 💻 What is Code.org's developer experience and tooling?
+
+Our developer experience may be different to what you’ve seen in your past roles. Working closely with our product team, we ship to production every weekday and we have a variety of cadences - sometimes you are writing code that goes live that day; other times you are on longer term projects - and while we do follow 2-weekly sprints, we do so in a lightweight way. We index towards getting the right things done the right way vs. taking a purist view of any particular methodology. [All our code](https://github.com/code-dot-org) is open source and we use JIRA and GitHub pull requests to track all of our work items. We also care a lot about quality and testing and are fortunate to have incredible support from [SauceLabs](https://saucelabs.com/), who help us run our suite of UI tests on various student devices (e.g., iPads) in their cloud device farm.
+
+#### 📟 What is the engineer on-call rotation at Code.org?
+
+Our 30+ engineering team shares on-call responsibilities. Each engineer covers a 24-hour shift and uses our escalation paths and playbooks to respond to site outages or other live-site issues. We are proud of our site’s reliability - on average, each engineer is only woken up once per year for issues that are typically quick to resolve. Instead, the "Developer of the Day" usually spends a portion of their workday managing the production deployment and triaging less urgent issues.
+
+#### 🏡 Are you a remote-friendly or work from home organization?
+
+Yes! In 2019, shortly before the pandemic, Code.org became a remote-first organization. Today, we have engineers in every US time zone and accommodate working hours accordingly. Most of our meetings happen between the hours of 10am and 3pm PT.
+
+We also recognize the value of in-person time, and every quarter we get together in person, typically in Seattle. Two of these events (we call them Kick-off week and “Codechella”) bring everyone in the organization together in January and June - and the engineering and product teams meet separately in April and October.
```

Comment:
Do we need to update the in-person event frequency?

Reply:
updated!